.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/astein/.pyenv/versions/fcli/bin/python"
```

Comment:
Please add this directory to `.gitignore`

Reply:
Bah! Every. Single. Time! I always do this when rushing it.